src/mlpack/methods/ann/loss_functions/poisson_nll_loss.hpp

```diff
@@ -111,8 +111,7 @@ class PoissonNLLLossType
 
  private:
   //! Check if the probabilities lie in the range [0, 1].
-  template<typename eT>
-  void CheckProbs(const arma::Mat<eT>& probs)
+  void CheckProbs(const MatType& probs)
```

Comment:
This is a nice improvement, I don't think there was ever a need for the `eT` parameter to begin with.